src/parser/index.js

```diff
@@ -0,0 +1,35 @@
+import getGitHubEventHeader from './getGitHubEventHeader';
+import parsePullRequestReviewFactory from './parsePullRequestReview';
+import parsePullRequestReviewComment from './parsePullRequestReviewComment';
+
+export default (client, logger) => {
+    const parsers = {
+        ping: null,
+        pull_request_review_comment: parsePullRequestReviewComment,
+        pull_request_review: parsePullRequestReviewFactory(client),
```

Comment:
Do you need to call the factory each time?
I would prefer `parsers[event](client)` and use it in factory only if needed.